src/tests/ftest/dfuse/bash_dcache.py

```diff
@@ -0,0 +1,82 @@
+"""
+  (C) Copyright 2024 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import os
+import stat
+
+from dfuse_test_base import DfuseTestBase
+from run_utils import run_remote
+
+SCRIPT = """#!/bin/bash
+
+#!/bin/bash
+
+dir_name="dir00"
+file_name="file00"
+
+rm -rf ${dir_name}
```

Comment:
This is a subprocess that removes a non-existant directory so will have no effect?

Reply:
rm with "-rf" returns 0 even the file/dir does not exist.

Reply:
It does, but it complicates the code for no reason?